pytorch_lightning/plugins/precision/precision_plugin.py

```diff
@@ -155,7 +155,14 @@ def optimizer_step(
     def _track_grad_norm(self, trainer: "pl.Trainer") -> None:
         if trainer.track_grad_norm == -1:
             return
-        kwargs = {"group_separator": trainer.logger.group_separator} if trainer.logger is not None else {}
+
+        if not trainer.loggers:
+            kwargs = {}
+        elif len(trainer.loggers) > 1:
+            kwargs = {"group_separator": trainer.loggers[0].group_separator}
+        else:
+            kwargs = {"group_separator": "/"}
```

Comment:
@akashkw I think you meant to flip lines 162 and 164?

Reply:
> @akashkw I think you meant to flip lines 162 and 164?

Whoops, that is correct. And the `"/"` is just the return value from `LightningLoggerBase.group_separator`, which would be called if `trainer.logger` were a `LoggerCollection`. Since we are deprecating `LoggerCollection` I just copied that return value here.